package.json

```diff
@@ -299,7 +299,7 @@
     "ts-retry-promise": "^0.6.0",
     "twilio": "^3.69.0",
     "typescript": "^5.4.3",
-    "typescript-json-schema": "^0.59.0",
+    "typescript-json-schema": "^0.64.0",
```

Comment:
i upgraded this because after updating some redux variables from `string` to `Address`, the inferred type in the schema became a string array (and this causes the `store` tests to fail, because the values are not actually arrays)